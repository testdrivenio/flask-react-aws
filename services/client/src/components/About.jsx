import React from 'react';

const About = () => (
  <div>
    <h1 className="title is-1">About</h1>
    <hr /><br />
    <p className="content">Add something relevant here.</p>
  </div>
)

export default About;
