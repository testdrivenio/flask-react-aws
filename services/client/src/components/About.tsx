import React from "react";
import { Box, Heading, Text, Divider, VStack } from "@chakra-ui/react";

const About = () => (
  <Box p={4} maxW="1200px" mx="auto" mt={12}>
    <VStack spacing={6} align="stretch">
      <Heading as="h1" size="2xl">
        About
      </Heading>
      <Divider />
      <Text fontSize="lg" className="content">
        Add something relevant here.
      </Text>
    </VStack>
  </Box>
);

export default About;
