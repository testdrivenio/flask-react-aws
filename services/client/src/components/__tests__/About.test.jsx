import React from "react";
import { render, cleanup } from "@testing-library/react";
import "@testing-library/jest-dom/extend-expect";

import About from "../About";

afterEach(cleanup);

it("renders properly", () => {
  const { getByText } = render(<About />);
  expect(getByText("Add something relevant here.")).toHaveClass("content");
});

it("renders", () => {
  const { asFragment } = render(<About />);
  expect(asFragment()).toMatchSnapshot();
});
