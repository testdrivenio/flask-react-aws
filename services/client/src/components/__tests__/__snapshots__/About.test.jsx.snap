// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders 1`] = `
<DocumentFragment>
  <div>
    <h1
      class="title is-1"
    >
      About
    </h1>
    <hr />
    <br />
    <p
      class="content"
    >
      Add something relevant here.
    </p>
  </div>
</DocumentFragment>
`;
